src/runtime/ExtendedNativeFunctionObject.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Escargot.h"
+#include "ExtendedNativeFunctionObject.h"
+
+namespace Escargot {
+
+void* ExtendedNativeFunctionObject::operator new(size_t size)
```

Comment:
`new` operator overloading is not necessary for this new class because all members are allocated on GC memory. We should overload the `new` operator when only there are some non-GC members and we want to annotate GC that these members do not need to be traced.